src/python/sdk/examples/trade.py

```diff
@@ -0,0 +1,372 @@
+from os import environ
+from typing import Tuple, Optional
+
+from aptos_sdk.account_address import AccountAddress
+from aptos_sdk.account import Account
+from aptos_sdk.client import RestClient, FaucetClient
+from aptos_sdk.transactions import EntryFunction, ModuleId
+from aptos_sdk.bcs import Serializer, encoder
+from aptos_sdk.type_tag import TypeTag, StructTag
+
+from econia_sdk.lib import EconiaViewer, EconiaClient;
+from econia_sdk.types import Side, SelfMatchBehavior, Restriction
+from econia_sdk.entry.market import change_order_size_user, cancel_all_orders_user, place_market_order_user_entry, place_limit_order_user_entry, register_market_base_coin_from_coinstore
+from econia_sdk.entry.user import register_market_account, deposit_from_coinstore
+from econia_sdk.view.market import get_price_levels
+from econia_sdk.view.registry import get_market_id_base_coin, get_market_registration_events
+from econia_sdk.view.resource_account import get_address
+from econia_sdk.view.user import get_change_order_size_events, get_market_account, get_place_limit_order_events, get_market_account
+
+"""
+If using a custom deployment...
+1. Run: cd /econia/src/move/econia (or whatever its full path is)
+2. Run: aptos init (recommended: press enter for all prompts, uses devnet)
+3. Run: export ECONIA_ADDR=<ADDR-FROM-ABOVE>
+4. Run: aptos move publish --override-size-check --included-artifacts none --named-addresses econia=$ECONIA_ADDR
+"""
+def get_econia_address() -> AccountAddress:
+    addr = environ.get("ECONIA_ADDR")
+    if addr == None:
+        # devnet default
+        return AccountAddress.from_hex("0xc0de0000fe693e08f668613c502360dc48508197401d2ac1ae79571498cd8b74")
+    else:
+        return AccountAddress.from_hex(addr)
+
+"""
+1. Run: cd /econia/src/move/faucet (or whatever its full path is)
+2. Run: aptos init (recommended: press enter for all prompts, uses devnet)
+3. Run: export FAUCET_ADDR=<ADDR-FROM-ABOVE>
+4. Run: aptos move publish --named-addresses econia_faucet=$FAUCET_ADDR
+"""
+def get_faucet_address() -> AccountAddress:
+    addr = environ.get("FAUCET_ADDR")
+    if addr == None:
+        print("Please provide the address of an Econia faucet as the FAUCET_ADDR environment variable.")
+        exit()
+    else:
+        return AccountAddress.from_hex(addr)
+    
+def get_aptos_node_url() -> str:
+    url = environ.get("APTOS_NODE_URL")
+    if url == None:
+        return "https://fullnode.devnet.aptoslabs.com/v1" # devnet default
+    else:
+        return url
+    
+def get_aptos_faucet_url() -> str:
+    url = environ.get("APTOS_FAUCET_URL")
+    if url == None:
+        return "https://faucet.devnet.aptoslabs.com" # devnet default
+    else:
+        return url
+
```

Comment:
Whitespace is inconsistent

Use a linter that formats on save

Correct this by using the formatter at `econia/scripts.sh` with the format code call

Reply:
ran the script just now that the diff is ready to publish